Baymax/Logger/LogFile.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  LogFile.swift
+//  Baymax
+//
+//  Created by Simon Mitchell on 01/04/2020.
+//  Copyright © 2020 3 SIDED CUBE. All rights reserved.
+//
+
+import Foundation
+
+/// A simple representation of a log file
+struct LogFile {
+    
+    /// The file's url
+    let url: URL
+    
+    /// The date the file was created
+    let creationDate: Date
```

Comment:
Only qualm I'd have with this is reading from disk might cause `UITableView` to lag a little bit if we're computing this every time we render a cell? I guess maybe not as it might not need to read the whole file in? I'll change 😂 (Talked myself out of it!)

Reply:
Actually I'm going back on myself again sorry! It'll require two calls to `resourceValues(forKey:)`. I think we're best leaving as-is for now ☺️ I don't think it's a dealbreaker if file sizes change as you're scrolling the view but don't update in the UI particularly!